modules/installation-about-disconnected.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-about-disconnected_{context}"]
+= About disconnected installations
+
+In {product-title} {version}, you can perform an installation that does not
+require an active connection to the internet to obtain software components. You
+complete a disconnected installation on only infrastructure that you provision,
+not infrastructure that the installer provisions, so your platform selection is
+limited.
+// maybe point out that you can follow the bare metal installation rules on supported hardware and link to the matrix
+
+If you choose to perform a disconnected installation on a cloud platform, you
+still require access to its cloud APIs and will still require internet access
+to use your cluster. Depending on your network, you might require less internet
+access for an installation on bare metal hardware or on VMware vSphere.
+
+To complete a disconnected installation, you must create a repository that
+mirrors the contents of the {product-title} registry and contains the
+installation media. You can create this mirror on a bastion host, which can
+access both the internet and your closed network, or by using other methods
+that meet your restrictions.
```

Comment:
Ah, here you're softening your [earlier bastion requirement][1].  I'd move some of this wording up to the prereq section, so folks don't say "well, we can't use a bastion" and give up before getting down this far ;).

[1]: https://github.com/openshift/openshift-docs/pull/16678/#discussion_r325790509